pkg/controllers/management/auth/prtb_handler.go

```diff
@@ -226,12 +233,12 @@ func (p *prtbLifecycle) removeMGMTProjectScopedPrivilegesInClusterNamespace(bind
 	return nil
 }
 
+// Prior to 2.5, for every ProjectRoleTemplateBinding, the following ClusterRoleBindings and RoleBindings are created
+// in the management cluster
+//  1. PRTB.UID is the label key for a CRB, PRTB.UID=membership-binding-owner
+//  2. PRTB.UID is label key for the RB, PRTB.UID=membership-binding-owner
+//  3. PRTB.UID is label key for RB, PRTB.UID=prtb-in-cluster-binding-owner
 func (p *prtbLifecycle) reconcileLabels(binding *v3.ProjectRoleTemplateBinding) error {
```

Comment:
We should move this comment back into the body of the function, comments directly above a function should describe the function, this is more of a general note. This comment also applies to other functions in this package, like the reconcileBindings function